docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,294 @@
+# Create a SinkBinding object
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+This topic describes how to create a SinkBinding object.
+The SinkBinding resolves a sink as a URI, sets the URI in the environment
+variable `K_SINK`, and adds the URI to a subject using `K_SINK`.
+If the URI changes, the SinkBinding updates the value of `K_SINK`.
+
+In the examples below, the sink is a Knative Service and the subject is a CronJob.
+If you have an existing subject and sink, you can replace the examples with your
+own values.
+
+## Before you begin
+
+Before you can create a SinkBinding object:
+
+- You must have Knative Eventing installed on your cluster.
+- Optional: If you want to use `kn` commands with SinkBinding, install the `kn` CLI.
+
+
+## Optional: Choose SinkBinding namespace selection behavior
+
+The SinkBinding object operates in one of two modes: `exclusion` or `inclusion`.
+
+The default mode is `exclusion`.
+In exclusion mode, SinkBinding behavior is enabled for the namespace by default.
+To disallow a namespace from being evaluated for mutation you must exclude it
+using the label `bindings.knative.dev/exclude: true`.
+
+In inclusion mode, SinkBinding behavior is not enabled for the namespace.
+Before a namespace can be evaluated for mutation, you must
+explicitly include it using the label `bindings.knative.dev/include: true`.
```

Comment:
It seems weird that lines are breaking throughout in this PR, like I'm not sure why this line ends here at "must"? Not a big deal, just weird.

Reply:
My text editor breaks lines at the 80 character mark to minimize horizontal scrolling. I probably removed a few words from that line as well. Line breaks such as these will not show up on the site.